src/test/java/bit/javaoop/people/FixedBudgetManagerTest.java

```diff
@@ -0,0 +1,96 @@
+package bit.javaoop.people;
+
+import bit.javaoop.CompanyAppException;
+import bit.javaoop.CompanyAppExceptionMessage;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FixedBudgetManagerTest {
+
+    private Manager manager;
+    private AbstractEmployee employee;
+    private double testHiringBudget;
+
+    @BeforeEach
+    void init() {
+        testHiringBudget = 30_000;
+        manager = new FixedBudgetManager("Manager", 30_000, testHiringBudget);
+        employee = new Employee("E1", 1_000);
+    }
+
+    @Test
+    void testIfCanNotHireEmployeeWithTooBigSalary() throws CompanyAppException {
```

Comment:
Postawiłbym bardziej na nazywanie metod testowych poprzez używanie słowa "should", np. 
`shouldCanNotHireEmployeeWithTooBigSalary()`

Reply:
Zależy od przyjętej konwencji - moim zdaniem nazwa jest jasna i mówi co test robi.